src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,47 @@
+.calendar {
+  margin: 71.5px 149px;
+  font-size: 30px;
+  padding: $square-paddings;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $square-spacing;
+  width: $calendar-width;
+
+  &__day {
+    @include square($square-size);
+
+    background-color: $background-color;
+    text-align: center;
+    line-height: $square-size;
+    cursor: pointer;
+    border: 1px solid black;
+    transition:
+      transform 500ms,
+      background-color 500ms;
+
+    &:hover {
+      background-color: $background-color-hover;
```

Comment:
The variable `$background-color-hover` is used here but needs to be defined in your SCSS files to avoid compilation errors.